JG/JohnnyGuitarNVSE.cpp

```diff
@@ -508,6 +508,10 @@ extern "C" {
 		REG_CMD(ClearMediaLocationControllerOverride); // do not document
 		REG_CMD(GetCasinoWinnings);
 		REG_CMD(SetCasinoWinnings);
+		REG_TYPED_CMD(GetCasinoDeckTexture, String);
```

Comment:
New functions need to be registered at the bottom of the list, otherwise it shifts the opcodes of existing commands